transaction-coordinator/src/main/java/org/apache/kafka/coordinator/transaction/TransactionLogConfig.java

```diff
@@ -85,7 +85,7 @@ public final class TransactionLogConfig {
     public TransactionLogConfig(AbstractConfig config) {
         this.config = config;
         this.transactionTopicMinISR = config.getInt(TRANSACTIONS_TOPIC_MIN_ISR_CONFIG);
-        this.transactionsLoadBufferSize = config.getInt(TRANSACTIONS_LOAD_BUFFER_SIZE_CONFIG);
+        this.transactionLoadBufferSize = config.getInt(TRANSACTIONS_LOAD_BUFFER_SIZE_CONFIG);
```

Comment:
The config name is `TRANSACTIONS_LOAD_BUFFER_SIZE_CONFIG`, why we need this change?

Reply:
Just Align other variables which all without `s`.
Only one variable is missing `s` is error-prone. 
The other choice is we add s to all other variables. WDYT?

Reply:
Add the reason to the PR  main text..

Reply:
IMHO, I will let these properties match the CONFIG name is better, thus I will modify other properties, ie: `transactionTopicMinISR` -> ``transactionsTopicMinISR`, but I didn't know this change is nessary or not.
cc @chia7712

Reply:
the consistency LGTM, thanks @mingyen066 to fix that